src/Engine/Objects/Player.cpp

```diff
@@ -248,7 +250,11 @@ bool Player::CanCastSpell(unsigned int uRequiredMana) {
     return false;
```

Comment:
If we've modified this function to be a simple accessor, then `pAudioPlayer->PlaySound` doesn't belong here anymore. Move it to call site?